### PR TITLE
ENH relock on the regs

### PR DESCRIPTION
**Pull request by beckermr**
cc @jaimergp

### CODE REVIEW

File: .github/workflows/relock.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+name: relock
+
+on:
+  workflow_dispatch: null
+  schedule:
+    - cron:  '15 * * * *'
```
Comment (jaimergp) [inline .github/workflows/relock.yml:6]: Isn't this too often? Every 15 mins? 🤔 
I'd say once or twice a week is more than enough.

Comment (beckermr) [reply .github/workflows/relock.yml:6]: I have a ton of changes coming through to make this a lot smarter. Sorry to bump you early.

Comment (jaimergp) [reply .github/workflows/relock.yml:6]: Ah ok, no probs!

Comment (beckermr) [reply .github/workflows/relock.yml:6]: OK I pushed to main for testing. See relock_me.py in the autotick bot directory.

**Pull request merged**